pkg/database/postgresql_database.go

```diff
@@ -0,0 +1,51 @@
+// Copyright KubeArchive Authors
```

Comment:
How about renaming this file to postgresql? As we already are inside the `database` package I think there is no need to reiterate that.

Reply:
Done.